### DISCUSSION TITLE
GPS position on web interface and offline maps

### DISCUSSION THREAD

**Discussion by 5B4APV** [dl9rdz/rdz_ttgo_sonde#117] [TTGO firmware]
Hello, is it possible to display the GPS coordinates of the tracker (those that have a GPS module) on the web interface and/or the OLED?

Also, would it be possible to implement offline maps? Could a predefined area be saved on an SD card and then used when no internet connection is available?

These are just some things that I would have used in the past couple of days using the tracker.

**Comment by RegsJR** [comment]
> Also, would it be possible to implement offline maps? Could a predefined area be saved on an SD card and then used when no internet connection is available?

Offline maps offer Czech service, app called Mapy.cz. Perfect solution, download offline maps at home as a batch. It would be very helpful for the users without mobile Internet access like me. :)
Thanks to all who make DL9RDZ better. 🥇

**Comment by dl9rdz** [comment]
There is some (experimental) offline map support available in the Android app (rdzwx-go)
You need to install the apk file directly from here https://github.com/dl9rdz/rdzwx-go/releases/tag/v1.0.7

Requires maps in mapsforge format (vector maps), and documentation of that features is limited.
There is no map downloader included, so ideally you use some other app that also uses mapsforge map files (e.g. c:geo, aprsdroid, locus - but note that not all locus maps are mapsforge maps), and just let rdzwx-go access those.

You can customize the look of the map using themes, e.g. uploading Desert.zip from maiwolf.de or Elevate.zip from OpenAndroMaps to the app.

**Reply by pavolgaj** [reply]
I have a problem with changing the map theme. I tried to download mentioned desert.zip and locus theme (https://docs.locusmap.eu/doku.php?id=manual:advanced:map_tools:theming). Theme in app didn't change:(

Good source of offline maps is https://osm.paws.cz/. Locus map format is same as format required by your app (rdzwx-go).
